ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/trashCan/TrashCanInteractionState.java

```diff
@@ -126,10 +126,10 @@ public void toMessage(TrashCanInteractionStateMessage message)
 
    public void fromMessage(TrashCanInteractionStateMessage message)
    {
-      super.fromMessage(message.getState());
-
       getDefinition().fromMessage(message.getDefinition());
 
+      super.fromMessage(message.getState());
```

Comment:
State fromMessage needs to happen after definition fromMessage. This was a mistake which reared it's head in the new implementation.